tests/tests/test-mock-hrmp.ts

```diff
@@ -1289,3 +1297,153 @@ describeDevMoonbeam("Mock XCM - receive horizontal transfer", (context) => {
     expect(alithAssetZeroBalance.isNone).to.eq(true);
   });
 });
+
+describeDevMoonbeam("Mock XCM - receive horizontal transact", (context) => {
+  let transferredBalance;
+  let alith: KeyringPair;
+  let DescendOriginAddress;
+  let sendingAddress;
+
+  before("Should receive transact action with DescendOrigin", async function () {
+    const keyringEth = new Keyring({ type: "ethereum" });
+    alith = keyringEth.addFromUri(ALITH_PRIV_KEY, null, "ethereum");
+    const allones = "0x0101010101010101010101010101010101010101";
+    sendingAddress = allones;
+
+    const derivedMultiLocation: MultiLocation = context.polkadotApi.createType(
+      "MultiLocation",
+      JSON.parse(
+        `{\
+            "parents": 1,\
+            "interior": {\
+              "X2": [\
+                { "Parachain": 1 },\
+                { "AccountKey20": \
+                  {\
+                    "network": "Any",\
+                    "key": "${allones}"\
+                  } \
+                }\
+              ]\
+            }\
+          }`
+      )
+    );
+
+    let toHash = new Uint8Array([
+      ...new Uint8Array([32]),
+      ...new TextEncoder().encode("multiloc"),
+      ...derivedMultiLocation.toU8a(),
+    ]);
+
+    DescendOriginAddress = u8aToHex(context.polkadotApi.registry.hash(toHash).slice(0, 20));
+
+    transferredBalance = 1n * GLMR;
+
+    // We first fund parachain 2000 sovreign account
+    await createBlockWithExtrinsic(
+      context,
+      alith,
+      context.polkadotApi.tx.balances.transfer(DescendOriginAddress, transferredBalance)
+    );
```

Comment:
I think `context.createBlock` is the current working implementation on master.

```suggestion
    await context.createBlock(
context.polkadotApi.tx.balances.transfer(DescendOriginAddress, transferredBalance)
    );
```